sylt-parser/src/parser.rs

```diff
@@ -6,13 +6,22 @@ use std::path::{Path, PathBuf};
 use sylt_common::error::Error;
 use sylt_common::rc::Rc;
 use sylt_common::Type as RuntimeType;
-use sylt_tokenizer::{file_to_tokens, Token};
+use sylt_tokenizer::{PlacedToken, Token, file_to_tokens};
 
 pub mod expression;
 pub mod statement;
 pub use self::expression::{Expression, ExpressionKind};
 pub use self::statement::{Statement, StatementKind};
 
+pub use sylt_tokenizer::Span;
+
+static INVALID_TOKEN: Token = Token::EOF;
+static INVALID_SPAN: Span = Span {
+    line: 0,
+    col_start: 0,
+    col_end: 0,
+};
```

Comment:
This is called both `INVALID_SPAN` and `zero`-span. But they have no difference. I suspect this should be called `ZERO_SPAN`, but you already have a function for that.

Reply:
:+1:. See https://github.com/FredTheDino/sylt-lang/pull/275#discussion_r663850706 for why the static is needed.